src/utils/round.ts

```diff
@@ -0,0 +1,5 @@
+export const round = (number: number, digits = 0): number => {
+  // We wrap `toFixed` call with `Number()` to remove insignificant trailing zeros
+  // because `(1).toFixed(2) // 1.00`
+  return Number(number.toFixed(digits));
```

Comment:
This doesn't seem very efficient. Converting a number to string back and forth means more memory allocations. Considering that the conversion usually happens during a UI interaction, this could result in worse performance. How about:

```js 
export const round = (number: number, digits = 0, base = Math.pow(10, digits)): number => {
  return Math.floor(base * number) / base
}
```

Reply:
toFixed doesn't convert a number to a string. It converts the number to a number. Just moving the point and rounds a number.

Reply:
Oh shit, you're right. It returns a string...

Reply:
@omgovich Unfortunately, it does:
```js
typeof (123.234).toFixed(1)
"string"
```

I agree the method name is absolutely confusing 😄

Reply:
Btw, the `Math.pow` implementation isn't ideal: it'll break when `digits` reaches the number overflow. But that's fine unless we plan to use the method externally.

Reply:
![image](https://user-images.githubusercontent.com/206567/96692603-e567c900-138e-11eb-8e77-bdf0835db1f5.png)

Your solution is way faster, thx!